### ISSUE TITLE
Confirm component show disable buttons when rendering more than once.

### ISSUE THREAD

**Issue by WiXSL** [marmelab/react-admin#3645]
Is related to [#3241](https://github.com/marmelab/react-admin/issues/3241)

[Example to reproduce it](https://codesandbox.io/s/tutorial-yfkgm)

1.- Check any row in the Posts List.
2.- Click the bulk action button.
3.- Confirm the dialog.
4.- Click the bulk action button again.

The problem is the internal state prop "loading" that is set to true handling confirmation.
Since "loading" is true, after closing the dialog, when you open it again, it disabled the buttons.

As a solution I set it to false and problem solved.

```js
// ...
class Confirm extends Component {
    state = { loading: false };

    handleConfirm = e => {
        e.stopPropagation();
        this.setState({ loading: true }); // <-- Just changing true to false here makes it work.
        this.props.onConfirm();
    };
 // ...
```

**Comment by fzaninotto** [comment]
This never happened to me because whenever a bulk action is confirmed, I unselect all the rows of the datagrid, which unmounts the bulk action button. The next time someone selects a row, the Confirm component mounts again, with an initial loading state of false. 

So unless you have a good reason not to unselect all rows after confirming, the problem is on your side.

**Comment by fzaninotto** [comment]
fixed by #3647
